packages/single-spa-welcome/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "scripts": {
+    "start": "webpack-dev-server",
+    "build": "webpack --mode=production",
+    "analyze": "webpack --mode=production --env.analyze=true",
+    "lint": "eslint src --ext js",
+    "format": "prettier --write './**'",
+    "test": "jest",
+    "watch-tests": "jest --watch",
+    "coverage": "jest --coverage"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "pretty-quick --staged && concurrently yarn:test yarn:lint"
+    }
+  },
+  "devDependencies": {
+    "@babel/core": "^7.7.5",
+    "@babel/plugin-transform-runtime": "^7.8.3",
+    "@babel/preset-env": "^7.7.6",
+    "@babel/preset-react": "^7.7.4",
+    "@babel/runtime": "^7.8.7",
+    "@testing-library/jest-dom": "^5.5.0",
+    "@testing-library/react": "^9.4.0",
+    "@types/jest": "^24.0.23",
+    "babel-eslint": "^11.0.0-beta.2",
+    "babel-jest": "^24.9.0",
+    "babel-loader": "^8.0.6",
+    "concurrently": "^5.0.1",
+    "css-loader": "^3.3.2",
+    "eslint": "^6.7.2",
+    "eslint-config-prettier": "^6.7.0",
+    "eslint-config-react-important-stuff": "^2.0.0",
+    "eslint-plugin-prettier": "^3.1.1",
+    "husky": "^3.1.0",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^24.9.0",
+    "jest-cli": "^24.9.0",
+    "prettier": "^2.0.4",
+    "pretty-quick": "^2.0.1",
+    "single-spa-react": "^2.14.0",
+    "style-loader": "^1.0.1",
+    "systemjs-webpack-interop": "^1.1.2",
+    "ts-config-single-spa": "^1.7.0",
```

Comment:
Hmm was this generated from create-single-spa? Looks like we are unintentionally installing this dependency even on javascript projects? I'll create an issue for this.

Reply:
https://github.com/single-spa/create-single-spa/issues/125

Reply:
Yes, this was a fresh install. I noticed this but forgot to log an issue.